namespace.go

```diff
@@ -153,6 +150,19 @@ func NamespaceWithEnvironmentBinding(name string) NamespaceOption {
 	}
 }
 
+func NamespaceWithAzureEnvironment(envName string) NamespaceOption {
```

Comment:
Add a comment explaining what this does, specifically what `envName` refers to.

Reply:
added doc comment